### PR TITLE
# Awesome books dynamic features

### PR DESCRIPTION
**Pull request by zewdu444**
# Awesome books dynamic features

- [x] files were added in the repo html, JavaScript and readme
- [x]  html element were added
- [x] dynamic feature in JavaScript were implemented
- [x] local storage has been used
- [x] array of objects were used to populate data dynamically
- [x] add and remove function were implemented

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,10 @@ To run the project, execute the following command:
 - GitHub: [@zewdu444](https://github.com/zewdu444)
 - Twitter: [@zewdu444](https://twitter.com/zewdu444)
 - LinkedIn: [zewdu-erkyhun](https://www.linkedin.com/in/zewdu-erkyhun-081378b3/)
+
+👤 **Abdimulhin Adan**
+
+- GitHub: [@AbdimulhinYussuf3675](https://github.com/AbdimulhinYussuf3675)
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (rloterh) [inline README.md:127]: - [ ] OPTIONAL
Kindly ensure that the _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.

File: README.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ To run the project, execute the following command:
 - GitHub: [@zewdu444](https://github.com/zewdu444)
 - Twitter: [@zewdu444](https://twitter.com/zewdu444)
 - LinkedIn: [zewdu-erkyhun](https://www.linkedin.com/in/zewdu-erkyhun-081378b3/)
+
```
Comment (rloterh) [inline README.md:121]: -[ ] OPTIONAL
Kindly remember to add appropriate content under all blank sections/headings.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Zewdu & Adan,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯
- Object correctly added!
- Local storage is correctly implemented!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (zewdu444) [reply README.md:127]: @rloterh thank you for the feedback we will correct it.

File: README.md (hunk 2 above)
Comment (zewdu444) [reply README.md:121]: @rloterh thank you for the feedback we will correct it.

**Pull request merged**